specs/src/specs/header-sync.md

```diff
@@ -0,0 +1,47 @@
+# P2P Header Sync
```

Comment:
Is this Header Sync or Header Exchange? 

In the code everything is referred to as Header Exchange so i think the spec should be written as Header Exchange. 

The action is performs is syncing headers over the P2P network. 
```suggestion
# Header Exchange
```

Reply:
We decided to go with Sync over Exchange correct? I'll update my comments accordingly, but we did align that we were removing P2P since that is how the sync works, vs what it does.

Reply:
yes that is correct.